docs/user_guide/PXE_Boot.inc

```diff
@@ -150,7 +150,7 @@ Under OSTYPE-MAJORRELEASE.MINORRELEASE-ARCH you should find the files:
 * ``initrd.img``
 * ``vmlinuz``
 
-If these are not there then you must create the directories as needed and copy
+If these directories do not exist, then create the directories as needed and copy
```

Comment:
The 'these' refers to the directory tree with the files.  If the files are not where they are supposed to be, you must create the directory tree and populate.